crypto/rand/rand_lib.c

```diff
@@ -281,7 +299,7 @@ int RAND_init_fips(void)
 
     FIPS_drbg_set_callbacks(dctx,
                             drbg_get_entropy, drbg_free_entropy, 20,
-                            drbg_get_entropy, drbg_free_entropy);
+                            drbg_get_nonce, drbg_free_entropy);
```

Comment:
Someone is going to double-take that drbg_free_entropy is paired with drbg_get_nonce, but the behavior should be fine.

Reply:
If you think it is confusing the users, I could fix it in one of the following ways:

- copy&paste `drbg_free_entropy` to `drbg_free_nonce`
- rename `drbg_free_entropy` to `drbg_free_entropy_or_nonce`

Please let me know what you prefer.

Reply:
I don't think any change is needed.